### PR TITLE
Graphical editor javadoc and code cleanups

### PR DESCRIPTION
**Pull request by philip-alldredge**
Graphical editor javadoc and code cleanups

### CODE REVIEW

File: ge/org.osate.ge/src/org/osate/ge/operations/StepResult.java [hunk 1]
```diff
@@ -92,6 +92,15 @@ public boolean aborted() {
 		return StepResultBuilder.create(userValue).build();
 	}
 
+	/**
+	 * Creates a new step result which indicates the step was completed but provides a null user value
+	 * @return a new step result with a null user.
```
Comment (RyanMcilnay) [inline ge/org.osate.ge/src/org/osate/ge/operations/StepResult.java:97]: add "value" after the word user

File: ge/org.osate.ge.gef/src/org/osate/ge/gef/LabelBackgroundColorUtil.java [hunk 2]
```diff
@@ -38,11 +38,12 @@ private LabelBackgroundColorUtil() {
 
 	/**
 	 * Sets the label background color by calling {@link HasLabelBackgroundColor#setLabelBackgroundColor(Color)} on the specified nodes.
+	 * @param <N> the type of nodes contained tin the specified node collection
```
Comment (RyanMcilnay) [inline ge/org.osate.ge.gef/src/org/osate/ge/gef/LabelBackgroundColorUtil.java:41]: tin needs to be in

File: ge/org.osate.ge.errormodel/src/org/osate/ge/errormodel/ui/palette/CreateErrorTypePaletteCommand.java [hunk 3]
```diff
@@ -35,7 +35,13 @@
 import org.osate.xtext.aadl2.errormodel.errorModel.ErrorModelFactory;
 import org.osate.xtext.aadl2.errormodel.errorModel.ErrorType;
 
+/**
+ * Palette command for creating error types.
```
Comment (RyanMcilnay) [inline ge/org.osate.ge.errormodel/src/org/osate/ge/errormodel/ui/palette/CreateErrorTypePaletteCommand.java:39]: use @link ErrorType here

**Review by RyanMcilnay — CHANGES_REQUESTED** [review]

File: ge/org.osate.ge.errormodel/src/org/osate/ge/errormodel/ui/palette/CreateErrorTypePaletteCommand.java (hunk 3 above)
Comment (philip-alldredge) [reply ge/org.osate.ge.errormodel/src/org/osate/ge/errormodel/ui/palette/CreateErrorTypePaletteCommand.java]: That was actually intentional but I adjusted it and similar ones to be consistent with the others.

**Pull request merged**